tests/bgp/test_bgp_authentication.py

```diff
@@ -0,0 +1,240 @@
+'''This script is to test the EBGP Authentication feature of SONiC.
+'''
+import logging
+
+import pytest
+import time
+
+from natsort import natsorted
+
+logger = logging.getLogger(__name__)
+
+bgp_config_sleeptime = 60
+bgp_pass = "sonic.123"
+mismatch_pass = "badpassword"
+
+
```

Comment:
What is the desired topo to run this test case? Can you add a topology marker for your test?

Reply:
I added a topology marker.